selene_sdk/samplers/online_sampler.py

```diff
@@ -82,13 +82,6 @@ class OnlineSampler(Sampler, metaclass=ABCMeta):
         documentation for `validation_holdout` for more details.
     sequence_length : int
         The length of the sequences to  train the model on.
-    bin_radius : int
```

Comment:
this might just be me being bad at git diff.... this isn't in our original onlinesampler right? at least when I look at the `master` branch it doesn't seem to be there so I'm confused why this is showing up in the diff ? just was trying to check if this is breaking change or not and it doesn't seem to be (just like you said in your comment)

Reply:
it's in the attributes https://github.com/FunctionLab/selene/blob/2150182c247d020da1baea7902e5add149053f84/selene_sdk/samplers/online_sampler.py#L85-L91 . I think I also changed all places that use these attributes.

Reply:
OH thanks, that makes a lot more sense haha